party/tests.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.test import APITestCase
+from rest_framework.test import APIClient
+from .models import Party
+
+
+class PartyAPI(APITestCase):
+
+    def setUp(self):
+        p = Party(name='Party', website='www.website.com')
```

Comment:
Nitpicking
Could you update this to `name='Party_name` or `name=`Framsókn`

Preferably "Framsókn" as we'll be dealing with special characters sooner or later